ui/src/testing/stubs/institution-api-stub.ts

```diff
@@ -5,30 +5,71 @@ import {
   GetInstitutionsResponse,
   GetPublicInstitutionDetailsResponse,
   Institution,
-  InstitutionApi,
+  InstitutionApi, InstitutionMembershipRequirement,
   OrganizationType,
 } from 'generated/fetch';
 import {stubNotImplementedError} from 'testing/stubs/stub-utils';
+import {
+  getRegisteredTierEmailAddresses, getRegisteredTierEmailDomains,
```

Comment:
nit: one per line

Reply:
done. I thought `yarn lint` fix would fix that.